### PR TITLE
udev: split files

### PR DESCRIPTION
**Pull request by yuwata**
No functional change, just splitting large files into small pieces.

### CODE REVIEW

File: src/libudev/libudev-queue.c [hunk 1]
```diff
@@ -192,18 +192,22 @@ _public_ struct udev_list_entry *udev_queue_get_queued_list_entry(struct udev_qu
...
-                return r;
+        fd = inotify_init1(IN_CLOEXEC);
+        if (fd < 0)
+                return -errno;
+
+        if (inotify_add_watch(fd, "/run/udev" , IN_DELETE) < 0)
+                return -errno;
 
-        return udev_queue->fd = r;
+        return udev_queue->fd = TAKE_FD(r);
```
Comment (ldv-alt) [inline src/libudev/libudev-queue.c]: It has to be `TAKE_FD(fd)` instead, and `r` becomes unused.

Comment (yuwata) [reply src/libudev/libudev-queue.c]: Ouch.

File: src/udev/udevd.h [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: GPL-2.0-or-later */
 #pragma once
 
-int run_udevd(int argc, char *argv[]);
+#include <stdbool.h>
```
Comment (bluca) [inline src/udev/udevd.h]: why is this include needed here?

File: src/udev/meson.build [hunk 3]
```diff
@@ -220,19 +220,19 @@ tests += [
                 'base' : test_libudev_base,
         },
         {
-                'sources' : files('test-udev-node.c'),
+                'sources' : files('test-udev-manager.c'),
                 'base' : test_libudev_base,
         },
         {
-                'sources' : files('test-udev-rules.c'),
+                'sources' : files('test-udev-node.c'),
```
Comment (bluca) [inline src/udev/meson.build:226]: looks like these renames are in the wrong commit

File: src/udev/udev-worker.c [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+#include "string-util.h"
+#include "udev-event.h"
+#include "udev-spawn.h"
+#include "udev-trace.h"
+#include "udev-util.h"
+#include "udev-watch.h"
+#include "udev-worker.h"
+
+void udev_worker_done(UdevWorker *worker) {
+        assert(worker);
```
Comment (bluca) [inline src/udev/udev-worker.c:28]: return early instead?

File: src/udev/meson.build (hunk 3 above)
Comment (ldv-alt) [reply src/udev/meson.build:226]: Not really, the source entries are just kept sorted.

Comment (bluca) [reply src/udev/meson.build:226]: github getting confused then

File: src/udev/udev-worker.c (hunk 4 above)
Comment (yuwata) [reply src/udev/udev-worker.c:28]: This is not free function, and we usually set assertion for `xyz_done()`. See e.g. `manager_varlink_done()` or so.

File: src/udev/udevd.h (hunk 2 above)
Comment (yuwata) [reply src/udev/udevd.h]: Ouch. I mixed in something wrong way. Resolved.

**Comment by yuwata** [comment]
@bluca Thank you for the review. The `stdbool.h` thing is fixed now. PTAL.

**Review by bluca — APPROVED** [review]

**Pull request merged**